src/app/crud/crud.component.html

```diff
@@ -1 +1,63 @@
-<p>crud works!</p>
+<section
+  class="container d-block w-50 h-50 p-1 bg-secondary text-white rounded">
+  <h6 class="text-center">Flight Booker</h6>
+  <div class="container d-block justify-content-center align-items-center p-3 bg-white text-dark w-100 rounded">
+    <div id="search">
+        <div class="row mb-2">
+            <label class="col-sm-2 col-form-label">Filter prefix:</label>
+            <div class="col-sm-3">
+              <input type="text" class="form-control" [(ngModel)]="search" (input)="find()" >
```

Comment:
Think how you can completely remove the usage of ngModel. Consider that it is banned :D